packages/ubuntu_desktop_installer/lib/pages/keyboard_layout_page.dart

```diff
@@ -47,19 +49,20 @@ class _KeyboardLayoutPageState extends State<KeyboardLayoutPage> {
         if (_selectedLayoutIndex > -1) {
           _selectedVariantIndex = _selectedLayout!.variants!
               .indexWhere((variant) => variant.code == info.variant);
-          if (_selectedVariantIndex == -1) {
-            _selectedVariantIndex = 0;
```

Comment:
is it intentional that the default variant is no longer selected? the above `indexWhere` is comparing empty to null.

![Screenshot from 2021-07-14 07-44-00](https://user-images.githubusercontent.com/140617/125569265-f2b1d613-3efc-41cc-a067-fdee11e8c969.png)

Reply:
Fix proposal: https://github.com/canonical/ubuntu-desktop-installer/pull/168